haiku-games/bong/bong-1.4~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="BeOS version of Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS, based on the old arcade classic Pong"
```

Comment:
We like to have lines smaller than 80 chars:
```diff
-DESCRIPTION="Bong is a FreeWare game for BeOS, based on the old arcade classic Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS, based on the old arcade classic \
+Pong"
```

Please ignore my previous message (which I have since deleted, because it was wrong).
I though we did not need to pass `MACHINE=BePC CC=g++ LD=g++` to `make`, but this was true **_only_** on x86_gcc2. On x86_64, it fails to build if we remove these extra parameters.

This means that your PR will be OK once you fix the wrapping of `DESCRIPTION` and add double quotes to srcGitRev, as @korli suggested. (These quotes are not necessary, but we like to have them because we already use double quotes for the srcGitRev on all our recipes.)

I'm adding this here so we don't forget about this.
```diff
-srcGitRev=906d822b79bdae9328fe743dffe2871f96a30e0e
+srcGitRev="906d822b79bdae9328fe743dffe2871f96a30e0e"
```
Thanks for your update :)